src/include/cache.h

```diff
@@ -197,10 +197,6 @@ struct __wt_cache {
     uint64_t las_insert_count; /* Count of inserts to lookaside */
     uint64_t las_remove_count; /* Count of removes from lookaside */
```

Comment:
Apologies for not mentioning it in the earlier comment. I think `las_remove_count` is not used anywhere now. it may be better to remove it?

Reply:
I think it is. It currently is used in `wt_las_remove_block` which is not part of this change AFAICT.

Reply:
The function `wt_las_remove_block` is already removed as part of the code changes from `relevant_history` project. Now whenever a page gets instantiated, the lookaside contents are not brought back into memory and they stay in the lookaside.

Reply:
Okay, then it should be removed. I'm wondering why the scripts that check for unused fields didn't flag it. Maybe those check for other unused things but not fields.

Reply:
Yes, I think they check for unused functions, and maybe unused defines.